poet/src/build_project/mod.rs

```diff
@@ -64,8 +66,16 @@ fn render_document<'render>(
         syntax_set,
     }: ContentDocumentRenderingContext<'render>,
 ) -> Result<String> {
+    let filtered_authors: Vec<Author> = front_matter
```

Comment:
That can be done at `build_project` context.

Also `authors.get(basename)` returrns `Option<Author>`, so if author is missing/invalid it will just ignore it. There neeeds to be some validation.